sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/_utils.py

```diff
@@ -25,16 +26,16 @@
 
 def _get_sdk_version_prefix():
     is_on_app_service = "WEBSITE_SITE_NAME" in environ
-    is_attach_enabled = getenv("ApplicationInsightsAgent_EXTENSION_VERSION") == "~3"
+    is_attach_enabled = isdir("/agents/python/")
```

Comment:
`/agents/python` is expected to be from the root directory?

Reply:
Yes. And I confirmed that this function works correctly in App Service from any folder.